tests/js/jest.config.js

```diff
@@ -17,10 +17,6 @@ module.exports = {
 	setupFiles: [
 		'<rootDir>/tests/js/setup-globals',
 		'jest-localstorage-mock',
-		// Note that `element-internals-polyfill` was introduced to polyfill `HTMLElement.attachInternals()`,
-		// due to JSDom not supporting it. It can probably be removed when JSDom does implement support.
-		// See https://github.com/jsdom/jsdom/issues/3444.
-		'element-internals-polyfill',
```

Comment:
Why are we removing this? We're not upgrading or removing JS DOM, right?

Reply:
As far as i understand this package, it is needed for custom web elements that we had in Material 3, but now we don't use them anymore, thus we can remove it. Anyway, removing this doesn't change anything in js tests which all pass without any change needed.